java-client/src/main/java/org/apache/pegasus/client/retry/DefaultRetryPolicy.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.pegasus.client.retry;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+import org.apache.pegasus.client.ClientOptions;
+
+/**
+ * The default retry policy, which is the only policy before we introduce the retry policy
+ * mechanism. It only considers the timeout value to calculate the retry delay, that's why\ we need
```

Comment:
Why there is '\' after the 'why' word? Typo?

Reply:
> Why there is '' after the 'why' word? Typo?

it typo. i fix it.